addons/event_sale/tests/common.py

```diff
@@ -44,3 +46,66 @@ def setUpClass(cls):
             'date_end': fields.Datetime.to_string(datetime.today() + timedelta(days=15)),
             'date_tz': 'Europe/Brussels',
         })
+
+
+class TestEventSaleTicketWithPriceListCommon(TestEventSaleCommon):
+    @classmethod
+    def setUpClass(cls):
+        super(TestEventSaleTicketWithPriceListCommon, cls).setUpClass()
+        ResCurrency = cls.env['res.currency'].with_context({'active_test': False})
+        cls.usd_currency = ResCurrency.search([('name', '=', 'USD')])
+        cls.eur_currency = ResCurrency.search([('name', '=', 'EUR')])
```

Comment:
You could do only one search for both records, then filter the returned record set to get each record. It's an additional line instead of a db query.

Reply:
yes, a small gain of performance is always welcome